src/Razor/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/Legacy_3_1_LeadingWhiteSpace_WithComponent/TestComponent.codegen.cs

```diff
@@ -0,0 +1,54 @@
+// <auto-generated/>
+#pragma warning disable 1591
+namespace Test
+{
+    #line hidden
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Components;
+    public partial class TestComponent : Microsoft.AspNetCore.Components.ComponentBase
+    {
+        #pragma warning disable 1998
+        protected override void BuildRenderTree(Microsoft.AspNetCore.Components.Rendering.RenderTreeBuilder __builder)
+        {
+            __builder.OpenComponent<Test.SomeOtherComponent>(0);
+            __builder.AddAttribute(1, "ChildContent", (Microsoft.AspNetCore.Components.RenderFragment)((__builder2) => {
+                __builder2.AddMarkupContent(2, "\r\n    ");
```

Comment:
This line is an example of something that is stripped out in the 5.0 behavior but not in the 3.1 behavior.

There are many such other examples throughout the baselines. If you want to see the exact diff on the baselines representing the changes between 3.1 and 5.0 behaviors, it's the baseline changes in this commit: https://github.com/dotnet/aspnetcore/pull/28191/commits/f90208d7774cc1ac9c8413b43fcf4e88bdee0a1e